vagga.yaml

```diff
@@ -89,3 +99,13 @@ commands:
     children:
       app: *run-app
       db: *run-db
+
+  test: !Command
+    description: Run tests
+    container: test
+    run: pre-commit run --all-files
+
+  pre-commit: !Command
+    description: Run pre-commit
+    container: test
+    run: [pre-commit, run]
```

Comment:
I would expose just `[pre-commit]`, w/o `run` so it won't limit capabilities of running other subcommands